porfile.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfilo</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+
+        <nav id="navbar">
+            <a class="logo" href="#">My Logo</a>
+            <ul>
+                <li><a href="">Portoflio</a></li>
+                <li><a href="">My Contact</a></li>
+                <li><a href="">About</a></li>
+            </ul>
+        <img src="./icons8-hamburger-32 (1).png" alt="" class="hamburger">
+</nav>
+            
+<div class="eli">
+    <img src="img/Rectangle 6.svg" alt="Rectangle 6" id="Rectangle6">
+    <img src="img/Ellipse 11.svg" alt="Ellipse 11" id="Ellipse11">
+    <img src="img/Rectangle 7.svg" alt="Rectangle 7" id="Rectangle7">
+    <img src="img/Ellipse 5.svg" alt="Ellipse 5" id="Ellipse5">
+    <img src="img/Rectangle 4.svg" alt="Rectangle 4" id="Rectangle4">
+    <img src="img/Ellipse 3.svg" alt="Ellipse 3" id="Ellipse3">
+
+</div>
+          
+    <section id="headline">
+        <h1 class="title">I'm Samuel Ogoigbe</h1>
+        <h1 class="title">Glad to see you!</h1>
+        <p class="description">
+            I'm a software developer! I can help you build a product , feature or website Look through some of my work
+            and
+            experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+        </p>
+        <p class="contact-link">LET'S CONNECT</p>
+        <div class="svg-img">
+            
+              <a href="#"><img src="Assets/twitter.svg" alt="twitter link"></a>
+              <a href="#"><img src="Assets/Linkedin icon.svg" alt="linked link"></a>
+              <a href="#"><img src="Assets/medium.svg" alt="medium"></a> 
+              <a href="#"><img src="Assets/github.svg" alt="github link"></a>
+              <a href="#"><img src="Assets/angellist.svg" alt="doit"></a>
+            
+         </div>
```

Comment:
- [ Required ] I think it would be better if the `margins` and `paddings` of this part of your webpage are consistent with the `Figma` design, currently, the elements on your page appear to be too `close` to each other than they are in the `Figma` design. Please see the screenshot below for more details on this and rectify it.

## Figma design
<img src="https://user-images.githubusercontent.com/16392046/219794236-a53fd0ab-2bd9-466d-8763-c1dd7aa2540e.png" width="375px" />

## Your design
<img src="https://user-images.githubusercontent.com/16392046/219793748-3669b52d-1e12-42c9-a9de-3c2d203fb0cf.png" width="375px" />